src/raps/actions/depositCompound.js

```diff
@@ -72,7 +72,7 @@ const depositCompound = async (wallet, currentRap, index, parameters) => {
   let deposit = null;
   try {
     logger.sentry('[deposit] txn params', transactionParams);
-    const deposit = await compound.mint(rawInputAmount, transactionParams);
```

Comment:
we were assigning `deposit` to a new `const` here when it should have been reassigning the `deposit` we just created with `let` two lines above